podpac/core/data/datasource.py

```diff
@@ -275,6 +281,45 @@ def _get_data(self, rc, rci):
     # Methods
     # ------------------------------------------------------------------------------------------------------------------
 
+    def eval(self, coordinates, **kwargs):
+
+        # check for missing dimensions
+        for c in self.coordinates.values():
+            if isinstance(c, Coordinates1d):
+                if c.name not in coordinates.udims:
+                    raise ValueError("Cannot evaluate these coordinates, missing dim '%s'" % c.name)
+            elif isinstance(c, StackedCoordinates):
+                if all(s.name not in coordinates.udims for s in c):
+                    raise ValueError("Cannot evaluate these coordinates, missing at least one dim in '%s'" % c.name)
+
+        # store original requested coordinates
+        requested_coordinates = coordinates
+
+        # remove extra dimensions
+        extra = [
+            c.name
+            for c in coordinates.values()
+            if (isinstance(c, Coordinates1d) and c.name not in self.coordinates.udims)
+            or (isinstance(c, StackedCoordinates) and all(dim not in self.coordinates.udims for dim in c.dims))
+        ]
+        coordinates = coordinates.drop(extra)
+
+        # transform coordinates into native crs if different
+        if coordinates.crs.lower() != self.coordinates.crs.lower():
+            coordinates = coordinates.transform(self.coordinates.crs)
+
+        output = super().eval(coordinates, **kwargs)
```

Comment:
Again, please add a comment about why we are calling the super here instead of using `_eval`